sdk/computefleet/Azure.ResourceManager.ComputeFleet/CHANGELOG.md

```diff
@@ -1,14 +1,32 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-11-01)
 
+-  GA version
+
+### Features Added
+
+This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html), and provides many core capabilities:
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET.
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing.
+    - HTTP pipeline with custom policies.
+    - Better error-handling.
+    - Support uniform telemetry across all languages.
+
+## 1.0.0-beta.2 (2023-11)
```

Comment:
remove this entry since you already added a new GA entry

Reply:
Ok, will remove - ## 1.0.0-beta.2 (2023-11)